### ISSUE TITLE
Complete country name from the code like 'Australia' from 'AU'

### ISSUE THREAD

**Issue by ayraju** [carmen-ruby/carmen-rails#18]
How can we get complete country name from the code like 'Australia' from 'AU'?

Any help?

**Comment by jason-rutherford** [comment]
``` ruby
c = Carmen::Country.coded('AU')
 => <#Carmen::Country name="Australia"> 
c.name
 => "Australia" 
```

**Comment by ayraju** [comment]
@c= 'US'
@country = Carmen::Country.coded(@c)

yes I found this but forgot to update it .

Anyway thank you.

**Comment by lawrobi** [comment]
If we write this way in the controller

@c = 'current_user.country_code'
@country = Carmen::Country.coded(@c)

It doesn't work ...

Can you help?

**Comment by frandroid** [comment]
You say it doesn't work, but you don't say how. What do you get when you do 
p @country
in the console, or you print @country on your web page?  Surely some form of object got passed to @country...  :) Look at both jason's and raju's code to understand.

**Comment by lawrobi** [comment]
yes when I print @country there is nothing (blank)... even though country_zone = US for that user.

**Comment by frandroid** [comment]
My point is that what's returned is not a string, but a Country object.  Maybe you could invoke the object's .name method?
